content/en/docs/content-structure/layout-builder/_index.md

```diff
@@ -99,24 +113,24 @@ Sets of content with a headline, description, and link displayed in 2 to 4-item
 
 | Label                 | Machine Name                | Type             | Required | Cardinality | Help text                                            | Field Settings    | Notes                              |
 |-----------------------|-----------------------------|------------------|----------|-------------|------------------------------------------------------|-------------------|------------------------------------|
-| Section title         | (inherit)                   |
-| Section subtitle      | (inherit)                   |
-| Grid section CTA/link | field_grid_section_cta_link | Link             | no       | 1           | A link button to be displayed below the grid content |
-| # of columns          | field_columns               | List (text)      | yes      | 1           |                                                      |                   | Allows 2-4 columns, defaults to 4. |
-| Grid Items            | field_grid_items            | Entity Reference | yes      | unlimited   |                                                      | Bundle: Grid Item |
+| Section heading         | (inherit)                   |  |  |  |  |  | H2 |
+| Section subheading     | (inherit)                   |  |  |  |  |  | H3 |
+| Grid section CTA / link | field_section_link | Link             | no       | 1           | A link button to be displayed below the grid content |
+| # of columns          | field_section_columns               | List (text)      | yes      | 1           |                                                      |                   | Allows 2-4 columns, defaults to 4. |
+| Grid Items            | field_grid__cta_item            | Entity Reference | yes      | 4   |                                                      | Bundle: grid_cta_item |
```

Comment:
@froboy cardinality was set to unlimited, I changed it to 4 to match what we have for the card component.

Reply:
Removed extra space, and referenced the exact machine name for the bundled items in all instances.